.github/workflows/requirements-lint.in

```diff
@@ -0,0 +1 @@
+yamllint
```

Comment:
Is it possible to specify the desired yamllint version here?

Reply:
added.